<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>429</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>621</width>
    <height>429</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cooquiz</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_comecar">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>350</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>come√ßar</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_a">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>82</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>a)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_b">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>82</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>b)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_c">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>82</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>c)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_d">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>33</width>
      <height>17</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>82</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>d)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_enviar">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>350</y>
      <width>251</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>enviar resposta</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>621</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>330</y>
      <width>621</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_questao">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>601</width>
      <height>111</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;justify&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_alternativa_a">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>551</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_alternativa_b">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>551</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_alternativa_c">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>230</y>
      <width>551</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_alternativa_d">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>551</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>21</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>true</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_enviar</sender>
   <signal>clicked()</signal>
   <receiver>radioButton_d</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel">
     <x>37</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_enviar</sender>
   <signal>clicked()</signal>
   <receiver>radioButton_c</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>392</y>
    </hint>
    <hint type="destinationlabel">
     <x>32</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_enviar</sender>
   <signal>clicked()</signal>
   <receiver>radioButton_b</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>397</y>
    </hint>
    <hint type="destinationlabel">
     <x>31</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_enviar</sender>
   <signal>clicked()</signal>
   <receiver>radioButton_a</receiver>
   <slot>toggle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>389</y>
    </hint>
    <hint type="destinationlabel">
     <x>33</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
